pkg/api/dump.go

```diff
@@ -254,25 +257,47 @@ func (s *service) runcDump(ctx context.Context, root, containerID string, pid in
 
 	// CRIU ntfy hooks get run before this,
 	// so have to ensure that image files aren't tampered with
-	s.postDump(ctx, opts.ImagesDirectory, state, false)
+	s.postDump(ctx, opts.ImagesDirectory, state, false, nil)
 
 	return nil
 }
 
 func (s *service) containerdDump(ctx context.Context, imagePath, containerID string, state *task.ProcessState) error {
 	// CRIU ntfy hooks get run before this,
 	// so have to ensure that image files aren't tampered with
-	s.postDump(ctx, imagePath, state, false)
+	s.postDump(ctx, imagePath, state, false, nil)
 
 	return nil
 }
 
+func (s *service) setupStreamerCapture(dumpdir string) *exec.Cmd {
+	buf := new(bytes.Buffer)
+	cmd := exec.Command("sudo", "cedana-image-streamer", "--dir", dumpdir, "capture")
```

Comment:
this fails if cedana-image-streamer isn't in $PATH. I would check if it exists at the cmd level (where --stream is passed) and not progress if so